library/book.rb

```diff
@@ -0,0 +1,18 @@
+require './library/rental'
+
+# Book class
+
+class Book
+  attr_reader :rentals
```

Comment:
- [ ] The rentals need to have `attr_accessor` since they both have to read and write each rental

Reply:
Yes I missed that, thank you for your feedback 👍🏼

Reply:
This issue has been addressed well. Great job! :clap: :clap: :clap:

Reply:
Thank you @henatan99, @mke2111